src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,41 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> findAndPrintEurekaNumbers(long minNumber, long maxNumber) {
 
-		List<Long> result = new ArrayList<Long>();
+		List<Long> eurakeNumbers = new ArrayList<Long>();
 
-		for (long i = a; i < b; i++) {
+		for (long currentNumber = minNumber; currentNumber < maxNumber; currentNumber++) {
 
-			List<Long> longs = new ArrayList<Long>();
+			List<Long> currentNumberAsList = new ArrayList<Long>();
+			String currentNumberAsString = Long.toString(currentNumber);
```

Comment:
Could be inside splitStringToList since this is not needed inside the current method